cli/internal/fs/turbo_json.go

```diff
@@ -165,7 +171,20 @@ func (c *TaskDefinition) UnmarshalJSON(data []byte) error {
 	// from an empty array. We can't use omitempty because it will
 	// always unmarshal into an empty array which is not what we want.
 	if rawPipeline.Outputs != nil {
-		c.Outputs = *rawPipeline.Outputs
+		Inclusions := make([]string, 0)
```

Comment:
Go note: typically local variables are not capitalized

Also, while we haven't always been great at this, the recommendation from the style guide is to initialize slices where you aren't setting the size as `nil`. `len()` and `append()` will do the right thing with them. So this line would be `var inclusions []string`